README.rst

```diff
@@ -1,75 +1,21 @@
+the-new-hotness
+---------------
+
 `Fedmsg <http://fedmsg.com>`_ consumer that listens to `release-monitoring.org
 <http://release-monitoring.org>`_ and files bugzilla bugs in response (to
 notify packagers that they can update their packages).
 
-Set up your environment with::
-
-    $ sudo yum install rpm yum-utils
-
-    $ virtualenv my-env --system-site-packages
-    $ source my-env/bin/activate
+For additional information see `documentation <https://the-new-hotness.readthedocs.io/en/stable/>`_.
 
-    $ python setup.py develop
+Development
+^^^^^^^^^^^
 
-And then run it with::
-
-    $ fedmsg-hub
-
-It should pick up the the-new-hotness consumer and start running.
+Contributions are welcomed, check out `contribution guide <https://the-new-hotness.readthedocs.io/en/stable/dev-guide.html#contribution-guidelines>_`.
```

Comment:
s/welcomed/welcome/
s/contribution guide/the contribution guidelines/

Reply:
Fixed